apps/air-discount-scheme/backend/src/app/modules/flight/flight.controller.ts

```diff
@@ -47,10 +54,77 @@ import { HttpRequest } from '../../app.types'
 export class PublicFlightController {
   constructor(
     private readonly flightService: FlightService,
+    @Inject(forwardRef(() => DiscountService))
     private readonly discountService: DiscountService,
     private readonly nationalRegistryService: NationalRegistryService,
   ) {}
 
+  @Get('discounts/:discountCode/checkFlightStatus')
```

Comment:
Needs a more descriptive name for connection flights.

Reply:
Oh you're right.

What about instead of checkFlightStatus:

- isValidConnectionFlight
- isValid
- canConnect

Reply:
I chose `isValidConnectionFlight`